### PR TITLE
Complete the resume for the mvp lab

### PR DESCRIPTION
**Pull request by clearloop**
## Need to be filled

+ [x] Team Details
+ [x] Personal connection
+ [x] Work plan
+ [ ] Video template
+ [x] Links

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,17 +36,15 @@ We focus to solve the web development experience for backend developers and newb
...
+We currently have little idea about this.
 
-## Work on
-> If we do not fund this project, will your team still work on it? To what extent? *
-
-We will work on it also, but it will cost more times because we really need some outside supports now, to prove this project is meaningful and to encourage us working harder, for the next chapter of the internet.
-
++ Develop websites with our library for public service for free.
++ We are members of a large rust community in China, lots of rust developers in that community seems like to try our library.
++ We'd like to hold a contest for developing web pages with our library.
```
Comment (boundless-forest) [inline README.md:45]: Great idea.

File: README.md [hunk 2]
```diff
@@ -36,17 +36,15 @@ We focus to solve the web development experience for backend developers and newb
...
 
+We currently have little idea about this.
 
-## Work on
-> If we do not fund this project, will your team still work on it? To what extent? *
-
-We will work on it also, but it will cost more times because we really need some outside supports now, to prove this project is meaningful and to encourage us working harder, for the next chapter of the internet.
-
++ Develop websites with our library for public service for free.
++ We are members of a large rust community in China, lots of rust developers in that community seems like to try our library.
```
Comment (boundless-forest) [inline README.md]: lots of rust developers in that community try to apply rust to front domain dev. Our library would be a good choice for them.

---

is better?

Comment (clearloop) [reply README.md]: y, we can commit to this one pr together, and merge it later, then we create a new pr for the last small modifications.

----

If got some ideas, just modify it.

File: README.md [hunk 3]
```diff
@@ -36,17 +32,14 @@ We focus to solve the web development experience for backend developers and newb
...
 > What is your personal connection to the service & the user group you are planning to reach with your product? *
 
+We currently have little idea about this.
 
-## Work on
-> If we do not fund this project, will your team still work on it? To what extent? *
-
-We will work on it also, but it will cost more times because we really need some outside supports now, to prove this project is meaningful and to encourage us working harder, for the next chapter of the internet.
-
++ Develop websites with our library for public service for free.
```
Comment (boundless-forest) [inline README.md:43]: This item might be useless. Open source means free to use.

File: README.md (hunk 2 above)
Comment (boundless-forest) [reply README.md]: OK

File: README.md (hunk 3 above)
Comment (clearloop) [reply README.md:43]: One of my friends maybe can help us(like contacting some students in CAFA), we can develop a Gallery for artworks for college and popularize the lib for newbies.

I was trying to explain this...

**Comment by clearloop** [comment]
I just completed my modification in this pr, I'm going to cut the template video now.

---

The empty-lines of each chapter:  Currently is 2, free to modify to other numbers.

**Pull request merged**